include/tack/float4.h

```diff
@@ -6,6 +6,293 @@
 #include <x86intrin.h>
 
 namespace tack {
+enum blend_t : int32_t {
+    e_aaaa = 0,
+    e_baaa,
+    e_abaa,
+    e_bbaa,
+    e_aaba,
+    e_baba,
+    e_abba,
+    e_bbba,
+    e_aaab,
+    e_baab,
+    e_abab,
+    e_bbab,
+    e_aabb,
+    e_babb,
+    e_abbb,
+    e_bbbb,
+};
+
+enum swizzle_t : int32_t {
+    e_xxxx = 0,
```

Comment:
Also perhaps we should move this into its own file or change the clang format so a number of enums are spaced out evenly over a single line.